custom-recipes/timeseries-preparation-resampling/recipe.json

```
@@ -1,159 +1,243 @@
 {
-    "meta": {
-        "label": "Time series resampling",
-        "description": "Performs interpolations and extrapolations to align time series on a fixed sampling rate.",
-        "icon": "icon-dku-timeseries-resample icon-DKU_timeseries_resample",
-        "displayOrderRank": 1
+  "meta": {
+    "label": "Time series resampling",
+    "description": "Performs interpolations and extrapolations to align time series on a fixed sampling rate.",
+    "icon": "icon-dku-timeseries-resample icon-DKU_timeseries_resample",
+    "displayOrderRank": 1
+  },
+  "kind": "PYTHON",
+  "selectableFromDataset": "input_dataset",
+  "inputRoles": [
+    {
+      "name": "input_dataset",
+      "label": "Input time series",
+      "description": "Dataset containing your time series data.",
+      "arity": "UNARY",
+      "required": true,
+      "acceptsDataset": true,
+      "acceptsFolder": false
+    }
+  ],
+  "outputRoles": [
+    {
+      "name": "output_dataset",
+      "label": "Output time series",
+      "description": "Output dataset with equispaced time series.",
+      "arity": "UNARY",
+      "required": true,
+      "acceptsDataset": true
+    }
+  ],
+  "params": [
+    {
+      "type": "SEPARATOR",
+      "name": "separator_input",
+      "label": "Input parameters"
     },
-    "kind": "PYTHON",
-    "selectableFromDataset": "input_dataset",
-    "inputRoles": [
-        {
-            "name": "input_dataset",
-            "label": "Input time series",
-            "description": "Dataset containing your time series data.",
-            "arity": "UNARY",
-            "required": true,
-            "acceptsDataset": true,
-            "acceptsFolder": false
+    {
+      "name": "datetime_column",
+      "label": "Timestamp column",
+      "type": "COLUMN",
+      "columnRole": "input_dataset",
+      "allowedColumnTypes": [
+        "date"
+      ],
+      "mandatory": true
+    },
+    {
+      "name": "advanced_activated",
+      "label": "Long format",
+      "description": "If multiple time series are stored in the same column",
+      "type": "BOOLEAN"
+    },
+    {
+      "name": "groupby_column",
+      "label": "Column with identifier",
+      "type": "COLUMN",
+      "columnRole": "input_dataset",
+      "mandatory": false,
+      "description": "⚠️  Deprecated. Use the field 'Column with identifiers' instead",
+      "visibilityCondition": "model.advanced_activated && model.groupby_column"
+    },
+    {
+      "name": "groupby_columns",
+      "label": "Columns with identifiers",
+      "description": "Columns that identify the multiple time series - mandatory for long format",
+      "type": "COLUMNS",
+      "allowedColumnTypes": [
+        "tinyint",
+        "smallint",
+        "int",
+        "bigint",
+        "string",
+        "boolean"
+      ],
+      "columnRole": "input_dataset",
+      "mandatory": false,
+      "visibilityCondition": "model.advanced_activated && (!model.groupby_column)"
```

Here to solve the issue previously raised, we must do that: 
```suggestion
       "visibilityCondition": "model.advanced_activated && (model.groupby_column.length == 0 || !model.groupby_column)"
```

---

done in https://github.com/dataiku/dss-plugin-timeseries-preparation/pull/16/commits/a29c79bcfbe9a15d4e58ae05fd3ca008b2ad9fb6